feature/home/presentation/src/main/java/com/sanaa/presentation/screen/PeopleScreen.kt

```diff
@@ -0,0 +1,173 @@
+import com.sanaa.designsystem.design_system.component.novix_scaffold.NovixScaffold
+import com.sanaa.designsystem.design_system.component.top_bar.NovixTopBar
+import com.sanaa.designsystem.design_system.component.top_bar.TopBarClickableIcon
+import com.sanaa.designsystem.design_system.theme.NovixTheme
+import com.sanaa.designsystem.design_system.theme.Theme
+import com.sanaa.presentation.components.cards.ActorCard
+import com.sanaa.presentation.ui_state.ActorUiModel
+import com.sanaa.presentation.ui_state.PeopleScreenEffects
+import com.sanaa.presentation.ui_state.PeopleScreenInteractionListener
+import com.sanaa.presentation.ui_state.PeopleScreenUiState
+import com.sanaa.presentation.viewmodel.PeopleViewModel
+import kotlinx.coroutines.flow.collectLatest
+
+
+@Composable
+fun PeopleScreen() {
+    val viewModel: PeopleViewModel = viewModel()
+    val state = viewModel.state.collectAsStateWithLifecycle()
+    val navController = rememberNavController()
+
+    LaunchedEffect(Unit) {
+        viewModel.effect.collectLatest { effect ->
+            when (effect) {
+                is PeopleScreenEffects.NavigateBack -> navController.popBackStack()
+                is PeopleScreenEffects.NavigateToActorDetails -> navController.navigate("actorDetails/${effect.actorId}")
+            }
+        }
+    }
+
+    PeopleScreenContent(
+        state = state.value,
+        interactionListener = viewModel
+    )
+}
+
+@Composable
+fun PeopleScreenContent(
+    state: PeopleScreenUiState,
+    interactionListener: PeopleScreenInteractionListener,
+) {
+    NovixScaffold {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .navigationBarsPadding()
+        ) {
+            NovixTopBar(
+                leftContent = {
+                    TopBarClickableIcon(
+                        icon = painterResource(id = com.sanaa.presentation.R.drawable.arrow_left_04),
+                        onClick = interactionListener::onBackClick
+                    )
+                },
+                screenTitle = stringResource(id = com.sanaa.presentation.R.string.people),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .systemBarsPadding()
+            )
+
+            Box(
+                modifier = Modifier
+                    .weight(1f)
+                    .fillMaxWidth(),
+                contentAlignment = Alignment.Center
+            ) {
+                AnimatedContent(
+                    targetState = state.isLoading,
+                    transitionSpec = { fadeIn() togetherWith fadeOut() },
+                    contentAlignment = Alignment.Center
+                ) { loading ->
+                    if (loading) {
+                        NovixLoadingIndicator()
+                    } else {
+                        ActorList(
+                            people = state.people,
+                            onItemClick = interactionListener::onActorClick
```

Comment:
Here we have 3 cases first loading second when return empty list and last when return data, here we didn't handle empty case